src/main/java/org/javacord/bot/commands/DocsCommand.java

```diff
@@ -64,10 +67,11 @@ public void onCommand(TextChannel channel, String[] args) {
      * @param embed The embed to populate.
      * @param searchString A search string.
      */
-    private void populateMethods(DiscordApi api, EmbedBuilder embed, String searchString) {
+    private void populateMethods(DiscordApi api, EmbedBuilder embed, String searchString, boolean includeAll) {
         Map<String, List<JavadocMethod>> methods = new JavadocParser(api, JavadocParser.getLatestJavaDocs(api).join())
                 .getMethods().join().stream()
                 .filter(method -> method.getFullName().toLowerCase().contains(searchString.toLowerCase()))
+                .filter(method -> includeAll || !method.getPackageName().endsWith("internal"))
```

Comment:
I'd change this check to say "ends with `.internal` or contains `.internal.`" (also in this order).
This way we can have deeper package structure in `internal` and the bot will do the right thing.
```suggestion
                .filter(method -> {
                    String packageName = method.getPackageName();
                    return includeAll || !(packageName.endsWith(".internal") || packageName.contains(".internal."));
                })
```